source/camera/basic/capture_hdr_complete_settings.py

```diff
@@ -17,29 +17,38 @@ def _main():
     camera = app.connect_camera()
 
     print("Configuring the camera settings")
-    iris_setting = [17, 27, 27]
-    exposure_setting = [10000, 10000, 40000]
-    gain_setting = [1.0, 1.0, 2.0]
-    settings_collection = [camera.settings for _ in range(3)]
-    for i in range(len(settings_collection)):
-        settings_collection[i].iris = iris_setting[i]
-        settings_collection[i].exposure_time = datetime.timedelta(microseconds=exposure_setting[i])
-        settings_collection[i].brightness = 1
-        settings_collection[i].gain = gain_setting[i]
-        settings_collection[i].bidirectional = 0
-        settings_collection[i].filters.contrast.enabled = True
-        settings_collection[i].filters.Contrast.threshold = 0.5
-        settings_collection[i].filters.gaussian.enabled = True
-        settings_collection[i].filters.gaussian.sigma = 1.5
-        settings_collection[i].filters.outlier.enabled = True
-        settings_collection[i].filters.outlier.threshold = 5
-        settings_collection[i].filters.reflection.enabled = True
-        settings_collection[i].filters.saturated.enabled = True
-        settings_collection[i].blue_balance = 1
-        settings_collection[i].red_balance = 1
+
+    settings = zivid.Settings(
+        acquisitions=[
+            zivid.Settings.Acquisition(
+                aperture=8.0, exposure_time=datetime.timedelta(microseconds=10000), brightness=1.8, gain=1.0,
+            ),
+            zivid.Settings.Acquisition(
+                aperture=4.0, exposure_time=datetime.timedelta(microseconds=10000), brightness=1.8, gain=1.0,
+            ),
+            zivid.Settings.Acquisition(
+                aperture=4.0, exposure_time=datetime.timedelta(microseconds=40000), brightness=1.8, gain=2.0,
+            ),
+        ],
+    )
+    filters = settings.processing.filters
+    filters.noise.removal.enabled = True
+    filters.noise.removal.threshold = 10
+    filters.smoothing.gaussian.enabled = True
+    filters.smoothing.gaussian.sigma = 1.5
+    filters.outlier.enabled = True
+    filters.outlier.threshold = 5
+    filters.experimental.contrast_distortion.correction.enabled = True
+    filters.experimental.contrast_distortion.correction.strength = 0.4
+    filters.experimental.contrast_distortion.removal.enabled = False
+    filters.experimental.contrast_distortion.removal.threshold = 0.5
```

Comment:
You have a few mistakes here:
```
    filters.outlier.enabled = True
    filters.outlier.threshold = 5
```
should be 
```
    filters.outlier.removal.enabled = True
    filters.outlier.removal.threshold = 5
```
and you forgot to set the reflection filter.
Unfortunately Python allows stuff like this. Setting a member that doesn't exist merely creates that member, and doesn't give an error. A good way to check results is to print the Settings at the end.

Reply:
done